ls1intum/Artemis
# `Programming exercises`: Fix an issue with git-diff report generation if it does not exist already

### Checklist
#### General


#### Server
#### Changes affecting Programming Exercises

### Motivation and Context



### Description


### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Do one of these:
    - Create a programming exercise (and maybe wait for the build to finish)
    - Import a programming exercise (and maybe wait for the build to finish)
    - Locally: Delete the git-diff report of a programming exercise in the database
4. Go to the exercise detail page
5. The git-diff report should be (re)generated and shown at the bottom

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
Unchanged

### Screenshots


The git-diff stats that should be visible:
<img width="412" alt="grafik" src="https://user-images.githubusercontent.com/18218285/190834980-08cd80c6-6e0f-4ff6-aaa2-2195219ab942.png">
